source/prefab/Card.h

```diff
@@ -0,0 +1,83 @@
+                        if(state == OPEN){ 
+                            prefab::Collapse accordion(card_header, card_body, true, emp::to_string(card_base+ "_card_collapse"));
+                            collapse_link = accordion.GetLinkDiv();
+                            collapse_body = accordion.GetToggleDiv();
+                        }
+                        else{
+                            prefab::Collapse accordion(card_header, card_body, false, emp::to_string(card_base + "_card_collapse"));
+                            collapse_link = accordion.GetLinkDiv();
+                            collapse_body = accordion.GetToggleDiv();
+                        }
+                        if(showGlyphs){ // by default add glyphs to a collapsible card
+                        // TODO: Don't use float, just set it to align to the right (need to make icon decend from div first)
+                            prefab::FontAwesomeIcon up("fa-angle-double-up");
+                            prefab::FontAwesomeIcon down("fa-angle-double-down");
+                            card_header << up;
+                            card_header << down;
+                            up.AddClass("float-right btn-link collapse_toggle setting_heading");
+                            down.AddClass("float-right btn-link collapse_toggle setting_heading");
+
+                        }
+                        *this << collapse_link;
+                        *this << collapse_body;
+                        card_header.AddClass("collapse_toggle_card_header");
+                    }
+                    else{ // plain card with no toggle enabled
+                        *this << card_header;
+                        *this << card_body;
+                    }
+                }
+                template <typename T>
+                void AddHeaderContent(T val, bool link_content=false){
+                    if(link_content){ 
+                        // add bootstrap link properities to content (hover, underline, etc.), 
+                        // but does not have a target or href because it is assumed that
+                        // this content will control the card collapse, which is done in the
+                        // constructor. If you want the content to link somewhere else, 
+                        // specify that in the web element/div param val {and set link_content=true TODO: verify this}.
+                        web::Div btn_link;
+                        btn_link.SetAttr("class", "btn-link");
+                        card_header << btn_link << val;
+                    }
+                    else{
+                        card_header << val;
+                    }
+                }
+                template <typename T>
+                void AddBodyContent(T val){
+                    card_body << val;
+                }
+                // TODO: override operator<< so that it streams into body content
```

Comment:
This will be great! We’ll want to be sure to update docs and examples and the app

Reply:
added it as issue #345